Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -136,8 +136,11 @@ const StatusBar = React.createClass({
     // Provide an imperative API as static functions of the component.
     // See the corresponding prop for more detail.
     setHidden(hidden: boolean, animation?: StatusBarAnimation) {
-      animation = animation || 'none';
-      StatusBar._defaultProps.hidden.value = hidden;
+      StatusBar._defaultProps.hidden = hidden;
+      if (!animation) {
```

Comment:
Note: this input is a string so a non-strict inequality check is used. The inputs below are booleans so strict inequality is used so `false` will be honored.